gcloud/bigtable/table.py

```diff
@@ -139,6 +139,36 @@ def create(self, initial_split_keys=None):
         # We expect a `._generated.bigtable_table_data_pb2.Table`
         client._table_stub.CreateTable(request_pb, client.timeout_seconds)
 
+    def rename(self, new_table_id):
+        """Rename this table.
+
+        .. note::
+
+            This cannot be used to move tables between clusters,
+            zones, or projects.
+
+        .. note::
+
+            The Bigtable Table Admin API currently returns
+
+                ``BigtableTableService.RenameTable is not yet implemented``
+
+            when this method is used. It's unclear when this method will
+            actually be supported by the API.
```

Comment:
For comparison, here is the output of the other [error case][1]

```python
>>> exc
NetworkError()
>>> exc.code
<StatusCode.INVALID_ARGUMENT: 3>
>>> exc.details
"Error in field 'cluster_id' : Invalid id for collection cluster : Length should be between [6,30], but found 32 (gcloud-python-cluster-0123456789)"
```

[1]: https://github.com/grpc/grpc/issues/2611#issuecomment-159721031